src/jl_uv.c

```diff
@@ -830,6 +830,51 @@ JL_DLLEXPORT int jl_tty_set_mode(uv_tty_t *handle, int mode)
     return uv_tty_set_mode(handle, mode);
 }
 
+typedef int (*work_cb_t)(void *, void *);
+typedef void (*notify_cb_t)(int);
+
+struct work_baton {
+    uv_work_t req;
+    work_cb_t work_func;
+    void      *work_args;
+    void      *work_retval;
+    notify_cb_t notify_func;
+    pid_t     tid;
```

Comment:
I was looking at linux travis logs. Also AppVeyor. Didn't check osx travis logs, so might be clang not respecting the warning flag.

Looking at the mingw-w64 headers it appears pid_t might be 64 bits on win64 with gcc? is it 32 bit int everywhere else?